File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <cctype>   // isdigit 헤더파일

using namespace std;

bool solution(string s) {
    bool answer = true;
    
    // 1. 문자열 길이 판별
    if(s.size() != 4 && s.size() != 6) {
        return false;
    }
    
    // 2. 숫자 문자열 판별
    for(char c : s) {
        if(!isdigit(c)) {
            return false;
        }
    }
    
    return answer;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {

        for(int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if(!Character.isDigit(c)) {
                return false;
            }
        }
        
        return s.length() == 4 || s.length() == 6;
    }
}